bool standing
bool facing_robot
bool waiving
bool crossed_hand